### ISSUE TITLE
Unable to import/use in angular 6 project

### ISSUE THREAD

**Issue by Moulde** [Kurtz1993/idle-service#3]
I attempted to use this library in a angular 6 project, but I was unable to import it in the library.
When i autoimport the Idleservice, i get the following import

    import { idleService } from '@kurtz1993/idle-service/dist/services/idle.service';

And when I run the project i get an error in core.js without any details.

I know this is not much detail :/

**Comment by Kurtz1993** [comment]
🤔 that's a weird one.
Have you tried importing from '@kurtz1993/idle-service' only?
Was that autoimport in VS Code?

**Comment by Moulde** [comment]
Yes, it was in vscode :) - It might have been because I attempted to import the IdleService (capital I).

I tried creating a small blank angular project, to test if my project had some bad configuration or something, but it seems the behavior is the same.
I am not exactly sure what causes the issues, but it does seem that I could import and run the project it after changing some imports, although the actual idle-timer never did anything, so I might have missed running a start method or something.

I think my issues was caused by a combination of the following:
- There is both a type IdleService, and an instance idleService, this seems very out of the ordinary. Usually you just import a type, and create an instance of it, in angular i guess this happens automatically when injecting the service in the constructor, but maybe this library's non-dependency to angular is why it has the const instance idleService?

- (i think) the readme has some inconsistencies, like, should the .on function not run on the "idleService" instance, instead of "service"?

Example
```typescript
import idleService, { IdleEvents } from '@kurtz1993/idle-service';

service.on(IdleEvents.UserIsBack, () => {
  console.log('User is back!');
});
```

With that said, I like the idea of the library not having a dependency to angular - although it would be really helpful with an example on how it can be used in angular.

Thanks
